gradle/libs.versions.toml

```
@@ -0,0 +1,51 @@
+[versions]
+hypertrace-grpcUtils = "0.9.0"
+
+guice = "5.1.0"
+grpc = "1.48.0"
```

```suggestion
grpc = "1.50.0"
```
(that's the version it'll get from grpcutils anyway)